community/ndp/kernelextension/src/main/java/org/neo4j/ext/NDPKernelExtension.java

```diff
@@ -100,7 +101,10 @@ public Lifecycle newKernelExtension( Dependencies dependencies ) throws Throwabl
 
         if ( config.get( Settings.ndp_enabled ) )
         {
-            final Sessions sessions = life.add( new StandardSessions( api, log ) );
+            final Sessions sessions = life.add( new ThreadedSessions(
+                    life.add( new StandardSessions( api, log ) ),
+                    life.add( new Neo4jJobScheduler() ),
```

Comment:
This shouldn't create a new scheduler here. Instead, add JobScheduler as a dependency in the Dependencies interface of this class, and we'll get the general database job scheduler from there.